### PR TITLE
chore: ci descr template

### PR DESCRIPTION
**Pull request by RafilxTenfen**
- Add confirmations to CI err msg
- Add PR template for new FP

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# New Finality Provider
```
Comment (vitsalis) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:1]: Should this be the template for all new PRs?

Comment (RafilxTenfen) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:1]: oh, no 
good catch, only for new fps

Comment (RafilxTenfen) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:1]: I will rename the file

Comment (RafilxTenfen) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Done at https://github.com/babylonchain/networks/pull/22/commits/0a869f34db4a763ba83a8a888e17885a0b784e44

**Review by vitsalis — APPROVED** [review]

**Pull request merged**